new-components/Dockerfile

```diff
@@ -26,14 +25,9 @@ RUN CGO_CPPFLAGS="-D_FORTIFY_SOURCE=2 -fstack-protector-all" \
 RUN mkdir -p /workspace
 
 # Used to actually run the binary in minimal image.
-FROM gcr.io/distroless/base-debian12:nonroot
+FROM gcr.io/distroless/base-debian12
```

Comment:
Removing nonroot due to this bug on Docker for volume mounts https://github.com/moby/moby/issues/2259